src/content/defi/index.md

```diff
@@ -5,348 +5,76 @@ lang: en
-
-Decentralized insurance aims to make insurance cheaper, faster to pay out, and more transparent. With more automation, coverage is more affordable and pay-outs are a lot quicker. The data used to decide on your claim is completely transparent.
-
-Ethereum products, like any software, can suffer from bugs and exploits. So right now a lot of insurance products in the space focus on protecting their users against loss of funds. However, there are projects starting to build out coverage for everything life can throw at us. A good example of this is Etherisc's Crop cover which aims to [protect smallholder farmers in Kenya against droughts and flooding](https://blog.etherisc.com/etherisc-teams-up-with-chainlink-to-deliver-crop-insurance-in-kenya-137e433c29dc). Decentralized insurance can provide cheaper cover for farmers who are often priced out of traditional insurance.
-
-<ButtonLink to="/dapps/?category=finance">
-  See insurance dapps
-</ButtonLink>
-
-<Divider />
-
-### Aggregators and portfolio managers {#aggregators}
-
-With so much going on, you'll need a way to keep track of all your investments, loans, and trades. There are a host of products that let you coordinate all your DeFi activity from one place. This is the beauty of DeFi's open architecture. Teams can build out interfaces where you can't just see your balances across products, you can use their features too. You might find this useful as you explore more of DeFi.
-
-<ButtonLink to="/dapps/?category=finance">
-  See portfolio dapps
-</ButtonLink>
-
-<Divider />
-
-## How does DeFi work? {#how-defi-works}
+You can think of DeFi in layers:
 
-DeFi uses cryptocurrencies and smart contracts to provide services that don't need intermediaries. In today's financial world, financial institutions act as guarantors of transactions. This gives these institutions immense power because your money flows through them. Plus billions of people around the world can't even access a bank account.
+1. The blockchain - Ethereum contains the transaction history and state of accounts
+2. The assets - ETH and other tokens stored on Ethereum
+3. The protocols - smart contracts thats provide the functionality, for example, decentralized lending of assets
+4. The applications - user interfaces thats are used to interact with the protocols
 
-In DeFi, a smart contract replaces the financial institution in the transaction. A smart contract is a type of Ethereum account that can hold funds and can send/refund them based on certain conditions. No one can alter that smart contract when it's live – it will always run as programmed.
+## What can you do with DeFi? {#what-can-you-do-with-defi}
 
-A contract that's designed to hand out an allowance or pocket money could be programmed to send money from Account A to Account B every Friday. And it will only ever do that as long as Account A has the required funds. No one can change the contract and add Account C as a recipient to steal funds.
+### Payments {#payments}
```

Comment:
Along with payments, perhaps we consider a note about simply "holding"?

May sound silly, but I kinda see a cold storage wallet holding funds on a decentralized, open, public, censorship resistant and immutable ledger as a form of "finance"... ie, just simply holding an asset safely until ready to spend, like many do with banks.

It's probably too implicit to bother mentioning, but thought I'd note

Reply:
I think that's an interesting idea. I don't think its necessary, but not strong against it. If you want to suggest copy I can include :)